documentation/docs/25-build-and-deploy/55-single-page-apps.md

```diff
@@ -9,7 +9,9 @@ You can turn any SvelteKit app, using any adapter, into a fully client-rendered
 export const ssr = false;
 ```
 
-> In most situations this is not recommended: it harms SEO, tends to slow down perceived performance, and makes your app inaccessible to users if JavaScript fails or is disabled (which happens [more often than you probably think](https://kryogenix.org/code/browser/everyonehasjs.html)).
+> SPA mode is highly discouraged: it destroys performance by forcing two network round trips before rendering can begin; harms SEO by downranking your page which is likely to fail core web vitals, excluding search engines that don't render JS, and causing your site to receive less frequent updates from those that do; and, finally, it makes your app inaccessible to users if JavaScript fails or is disabled (which happens [more often than you probably think](https://kryogenix.org/code/browser/everyonehasjs.html)).
```

Comment:
This is too blunt - if you create an internal app you don't care about these things, we should clarify in which situations SPAs are discouraged. I also fear that the language might put people off ("destroy" etc) thinking "oh so SvelteKit is not for me thanks I'll look elsewhere to create my SPA then" which neither helps their web vitals nor us. In general I don't think that more drastic language will help people using this in less situations, instead we need to make this easier discoverable through for example warnings which link back to the documentation.

```suggestion
> For public-facing websites (i.e. not behind a login or only used internally at a company), SPA mode is highly discouraged. It slows down performance by forcing two network round trips before rendering can begin, which harms SEO by downranking your page which is more likely to fail core web vitals. It also excludes search engines that don't render JS, and causes your site to receive less frequent updates from those that do. Finally, it makes your app inaccessible to users if JavaScript fails or is disabled (which happens [more often than you probably think](https://kryogenix.org/code/browser/everyonehasjs.html)).
```

Reply:
I just pushed an update that uses a lot of the changes you suggested here. Your suggestion had "which" twice in once sentence and I wanted to avoid that. I reworded a couple of other things as well to tone things down and incorporate some of your ideas